q2_phylogeny/plugin_setup.py

```diff
@@ -41,13 +42,15 @@
 plugin.methods.register_function(
     function=q2_phylogeny.fasttree,
     inputs={'alignment': FeatureData[AlignedSequence]},
-    parameters={},
+    parameters={'n_threads': Int},
     outputs=[('tree', Phylogeny[Unrooted])],
     input_descriptions={
         'alignment': ('Aligned sequences to be used for phylogenetic '
                       'reconstruction.')
     },
-    parameter_descriptions={},
+    parameter_descriptions={
+        'n_threads': 'The number of threads.'
```

Comment:
Can you add a note about `-1`? I tested this locally and that value seems to work as expected.

Pulling the text from mafft's `n_threads`:

```
(Use -1 to automatically use all available cores)
```

Reply:
Can you also note the non-deterministic behavior when `FastTreeMP` is used, i.e. when `n_threads` is not equal to `1`? @gregcaporaso [requested that](https://github.com/qiime2/q2-phylogeny/issues/35#issuecomment-316770256) in the corresponding issue.